### PR TITLE
More optimizations for results, primarily for contact fields now

### PR DESCRIPTION
**Pull request by nicpottier**
Also adds a new index on contact_field and location_value combined to speed up location queries.

### CODE REVIEW

**Comment by ericnewcomer** [comment]
yup

File: temba/values/models.py [hunk 1]
```diff
@@ -380,6 +386,7 @@ def get_value_summary(cls, ruleset=None, contact_field=None, filters=None, segme
         # does our value exist?
         r = get_redis_connection()
         cached = r.get(key)
+        cached = None
```
Comment (norkans7) [inline temba/values/models.py]: it seems this should be removed

Comment (ericnewcomer) [inline temba/values/models.py]: ha!

**Comment by norkans7** [comment]
Yep

**Pull request merged**